<launch>
  <arg name="namespace" default="voliro"/>
  <arg name="odometry_topic" default="msf_core/odometry"/>
  <arg name="waypoint_file_name" default="waypoints_contact"/>
  <arg name="world_frame_name" default="world"/>

  <group ns="$(arg namespace)">
    <node name="local_planner" pkg="omav_local_planner" type="voliro_trajectories_node" output="screen" clear_params="true">
      <param name="waypoints_filename" value="$(find omav_local_planner)/resource/eval_traj_small2.txt" />
    </node>
  </group>

</launch>